### DISCUSSION TITLE
covid 19

### DISCUSSION THREAD

**Discussion by Jerry10413** [1101-GS4719-C/Python-Community-GS4719-C#996] [Q&A]
現在防疫中心公布在外可以不用戴口罩，進入特定室內場所也可以不用配戴，並且確診後只要休息5天就可以回去繼續上班上課，各位覺得這樣是好的嗎??我自己是非常不認同的，看這解封開放但是疫情並沒有趨緩，每天還是1、2萬例確診在增加，政府還是照樣解封，不知道這樣的用意為何，然後確診後休息五天根本就是不夠的，通常五天後塊篩也還會是陽性，就這樣回去工作不就會傳染給更多人嗎?真的不知道在想什麼.....
歡迎提出自己的想法

**Comment by ruthchen1020** [comment]
大陸的變種病毒看起來有夠可怕，很多國家都學到了要趕快禁大陸的飛機，台灣還在放寬防疫政策。
當初因為提早防疫所以躲過，但這次大家都不重視的感覺。

**Comment by peggy0329** [comment]
在還沒宣布新政策時，我一直以為政府會到跨年完才宣布戶外解封
因為聖誕節+跨年群聚一定又會造成一大波人確診
但沒想到政府那麼快就宣布戶外可以不戴口罩
因此目前的確診數我沒有很意外
感覺政府為了經濟有一點對疫情鬆懈了

**Comment by delia07152** [comment]
雖然現在可以在公共場所不戴口罩，但我覺得大部分的人對於防疫以及自身健康的意識還是蠻高的，至少路上的多數人還是會戴著口罩，不過政策的確是有點太鬆懈了！感覺放寬的速度可以再慢一點，尤其寒假過年快到了，萬一因為政策的鬆散而使得一波疫情又產生，那實則不太划算，愉快的過年卻要在家裡獨自度過，聽起來有點悲傷。

**Comment by 12Jesse24** [comment]
但其實說穿了COVID-19就只是前所未有的大規模流行性感冒，等多數民眾都有打疫苗後就可以漸漸放寬管制了，現在也有很多的國家開放通往大陸旅遊了，所以疫情沒有大家想像的這麼恐怖，希望人類加油繼續撐下去。
